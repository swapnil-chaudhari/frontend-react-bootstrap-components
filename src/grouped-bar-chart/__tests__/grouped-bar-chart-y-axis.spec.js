import React from 'react';
import renderShallow from 'render-shallow';
import GroupedBarChartYaxis from '../grouped-bar-chart-y-axis';

describe('<GroupedBarChartYaxis>', () => {
    describe('when it renders', () => {
        let component;

        beforeAll(() => {
            component = renderShallow(
                <GroupedBarChartYaxis
                    ticks={ [1, 2] }
                    ticksFormat={ v => v }
                    scale={ v => v }
                    width={ 100 }
                    height={ 200 }
                    xOffset={ 30 }
                    yOffset={ 20 }
                />
            ).output;
        });

        test('has proper markup', () => {
            expect(component).toMatchSnapshot();
        });
    });
});
