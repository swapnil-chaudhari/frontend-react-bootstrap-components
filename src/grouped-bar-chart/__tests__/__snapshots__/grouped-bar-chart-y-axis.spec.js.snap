// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GroupedBarChartYaxis> when it renders has proper markup 1`] = `
<g>
  <line
    className="GroupedBarChartYaxis-line"
    x1={30}
    x2={30}
    y1={20}
    y2={200}
  />
  <g>
    <line
      className="GroupedBarChartYaxis-tick"
      x1={0}
      x2={100}
      y1={1}
      y2={1}
    />
  </g>
  <g>
    <line
      className="GroupedBarChartYaxis-tick"
      x1={0}
      x2={100}
      y1={2}
      y2={2}
    />
    <g>
      <text
        className="GroupedBarChartYaxis-label"
        x={0}
        y={2}
      >
        2
      </text>
    </g>
  </g>
</g>
`;
